.github/workflows/ci.yml

```diff
@@ -0,0 +1,161 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+env:
+  rust-version: 1.60.0
+
+jobs:
+  build:
+    name: cargo build
+    runs-on: ubuntu-latest
```

Comment:
Should we also run this on macos-latest and windows-latest?  I have definitely seen `cargo build` failures that only showed up on one environment.

Reply:
The crates checked in this `build` job only target `wasm32-unknown-unknown`. So I thought only run on ubuntu is sufficient. And ubuntu is the fastest.